pages/index.vue

```diff
@@ -64,19 +65,22 @@ export default Vue.extend({
         case 'latestmedia': {
           const latestMediaSections = [];
 
-          const userViewsRequest = await $api.userViews.getUserViews({
-            userId: $auth.user?.Id
-          });
+          let userViews: BaseItemDto[] = store.state.userViews.views;
+
+          if (!userViews.length) {
+            await store.dispatch('userViews/refreshUserViews');
+            userViews = await store.state.userViews.views;
+          }
```

Comment:
Testing this still but some comments just from looking at the code:

Imo, this is not clean at all. I understand the concept of getters as a function that does some processing with the data before returning it, not only returning the data as-is. Thus, I think it's better to move this whole logic to the ``getUserViews`` getter than dooing this here.

There won't be repetition as well: it will be a matter of checking the length of the state data, dispacth in case it's 0 and return it afterwards.

Much more cleaner and reusable in the long term (don't think we will ever need to reuse this, but better to have good practices)